src/braket/ir/jaqcd/program_v1.py

```diff
@@ -59,40 +59,44 @@
 )
 from braket.schema_common import BraketSchemaBase, BraketSchemaHeader
 
-GateInstructions = Union[
-    CCNot,
-    CNot,
-    CPhaseShift,
-    CPhaseShift00,
-    CPhaseShift01,
-    CPhaseShift10,
-    CSwap,
-    CY,
-    CZ,
-    H,
-    I,
-    ISwap,
-    PhaseShift,
-    PSwap,
-    Rx,
-    Ry,
-    Rz,
-    S,
-    Swap,
-    Si,
-    T,
-    Ti,
-    Unitary,
-    V,
-    Vi,
-    X,
-    XX,
-    XY,
-    Y,
-    YY,
-    Z,
-    ZZ,
-]
+"""
+We need a constant lookup for our pydantic validator. Using a plain Union[] will result
```

Comment:
The pydantic validator requires a constant lookup function. A plain Union[] results
in an O(n) lookup cost for arbitrary payloads, which has a negative impact on model parsing times.

(Note to writer--active voice, present tense, eliminate "we" for AWS style)

Reply:
Hah, this change wasn't even mine; I screwed up some git commands. Who would've thought that screwup would lead to doc improvements